app/gui2/shared/languageServer.ts

```diff
@@ -21,6 +19,21 @@ import type {
 } from './languageServerTypes'
 import type { Uuid } from './yjsModel'
 
+const DEBUG_LOG_RPC = false
```

Comment:
I don't like ENVs for that purpose, because you need to kill and restart the server manually to set them. Normally I'd go with some kind of runtime config flags for that purpose, but actually IS part of the server code, so you can't toggle it from the UI anyway. That's why i chose to do the easiest possible option, and leave improving on that for another time/PR.